rep-0149.rst

```diff
@@ -1007,6 +1007,12 @@ Therefore the environment variable `ROS_VERSION` is used with the value being
 either `1` or `2`.
 This new environment variable can be defined beside the `ROS_DISTRO` one.
 
+An additional environment variable `ROS_PYTHON_VERSION` to choose the Python
+version in use (either `2` or `3`) is also provided.  In ROS 1, this allows
```

Comment:
Wrap before each sentence.

Reply:
Fixed in b53636244c08faa785d2647f738a868c34317fbb